xbmc/windowing/gbm/drm/DRMAtomic.cpp

```diff
@@ -267,6 +245,34 @@ bool CDRMAtomic::AddProperty(CDRMObject* object, const char* name, uint64_t valu
   return true;
 }
 
+bool CDRMAtomic::AddPropertyValueByName(struct drm_object* object,
```

Comment:
`drm_object` struct doesn't exist anymore. You'll have to use the `CDRMObject` class.

Reply:
Ok got it. I will fix it .